docs/sources/administration/correlations/correlation-configuration/index.md

```diff
@@ -0,0 +1,102 @@
+
+## Source data source and result field
+
+Links are shown in Explore visualizations for the results from the correlation’s source data source. A link is assigned to one of the fields from the result provided in the correlation configuration (the results field). Each visualization displays fields with links in a different way ([Correlations in Logs Panel]({{< relref "./use-correlations-in-visualizations#correlations-in-logs-panel">}}) and see [Correlations in Table]({{< relref "./use-correlations-in-visualizations#correlations-in-table">}}))
+
+## Target query
+
+The target query is run when a link is clicked in the visualization. You can use the query editor of the selected target data source to specify the target query. Source data results can be accessed inside the target query with variables.
+
+### Correlation Variables
+
+You can use variables inside the target query to access the source data related to the query. Correlations use [Grafana variable syntax]({{< relref "/docs/grafana/latest/dashboards/variables/variable-syntax" >}}). Variables are filled with values from the source results when the link is clicked. There are two types of variables you can use:
+
+- [field variables]({{< relref "/docs/grafana/latest/panels-visualizations/configure-data-links#field-variables" >}}) (allows to access field values and labels)
+- correlation variables (allows to access field values and transformations)
+
+Example: If source results contain a field called “employee”, the value of the field can be accessed with:
+
+- A field variable ${\_\_data.fields.employee}
+- A correlation variable that maps the field value above to ${employee}
+
+Additionally to mapping field values to shorter variables names, more correlation variables can be created by applying transformations to existing fields.
+
+Please check an example in [Use variables and transformations in a correlation]({{< relref "./use-variables-and-transformations" >}}).
+
+### Correlation Transformations
+
+Correlations provide a way to extract more variables out of field values. The output of transformations is a set of new variables that can be accessed as any other variable.
+
+There are two types of transformations: logfmt and regular expression.
+
+Each transformation uses a selected field value as the input. The output of a transformation is a set of new variables based on the type and options of the transformation.
+
+Please check an example in [Use variables and transformations in a correlation]({{< relref "./use-variables-and-transformations" >}}) for more details.
+
+### Logfmt transformation
+
+Logfmt transformation deconstructs a field value containing text formatted with [logfmt key/value pairs](https://brandur.org/logfmt). Each pair becomes a variable with the key being the name of the variable.
+
+Logfmt transformation requires only specifying the input field name.
```

Comment:
The logfmt transformation only requires specifying the input field name if you would like the transformation to apply to a different field than the results field.

Reply:
Updated in 44ba3663431b9c01d65eb636dc3e171ad70a291d.